src/dtos/http_response.go

```diff
@@ -0,0 +1,58 @@
+package dtos
+
+import (
+	"time"
+
+	"github.com/Real-Dev-Squad/wisee-backend/src/models"
+)
+
+type ErrorResponse struct {
+	Message string      `json:"message"`
+	Detail  interface{} `json:"detail"`
+}
+
+type ResponseDto struct {
+	Message string        `json:"message"`
+	Data    interface{}   `json:"data"`
+	Error   ErrorResponse `json:"error"`
+}
+
+type CreateUpdateGetFormResponseDto struct {
+	Id          int64              `json:"id"`
+	Content     models.FormContent `json:"content"`
+	OwnerId     int64              `json:"owner_id"`
+	CreatedById int64              `json:"created_by_id"`
+	Status      string             `json:"status"`
+	UpdatedById *int64             `json:"updated_by_id"`
+	CreatedAt   string             `json:"created_at"`
+	UpdatedAt   string             `json:"updated_at"`
+}
+
+type GetFormMetaDataResponseDto struct {
```

Comment:
- will `FormMetaDataResponseDto` be a more suitable name here ?

Reply:
the naming im trying to use is [Operation][Table]ResponseDto,